{
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-19844"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)"
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-640"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/",
          "refsource": "CONFIRM",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/"
        },
        {
          "name": "https://docs.djangoproject.com/en/dev/releases/security/",
          "refsource": "MISC",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://docs.djangoproject.com/en/dev/releases/security/"
        },
        {
          "name": "USN-4224-1",
          "refsource": "UBUNTU",
          "tags": [
            "Third Party Advisory"
          ],
          "url": "https://usn.ubuntu.com/4224-1/"
        },
        {
          "name": "DSA-4598",
          "refsource": "DEBIAN",
          "url": "https://www.debian.org/security/2020/dsa-4598"
        },
        {
          "name": "20200108 [SECURITY] [DSA 4598-1] python-django security update",
          "refsource": "BUGTRAQ",
          "url": "https://seclists.org/bugtraq/2020/Jan/9"
        },
        {
          "name": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html",
          "refsource": "MISC",
          "url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html"
        },
        {
          "name": "https://security.netapp.com/advisory/ntap-20200110-0003/",
          "refsource": "CONFIRM",
          "url": "https://security.netapp.com/advisory/ntap-20200110-0003/"
        },
        {
          "name": "GLSA-202004-17",
          "refsource": "GENTOO",
          "url": "https://security.gentoo.org/glsa/202004-17"
        },
        {
          "name": "https://groups.google.com/forum/#%21topic/django-announce/3oaB2rVH3a0",
          "url": "https://groups.google.com/forum/#%21topic/django-announce/3oaB2rVH3a0"
        },
        {
          "name": "FEDORA-2020-adb4f0143a",
          "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD/"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "LOW",
        "accessVector": "NETWORK",
        "authentication": "NONE",
        "availabilityImpact": "NONE",
        "baseScore": 5,
        "confidentialityImpact": "NONE",
        "integrityImpact": "PARTIAL",
        "vectorString": "AV:N/AC:L/Au:N/C:N/I:P/A:N",
        "version": "2.0"
      },
      "exploitabilityScore": 10,
      "impactScore": 2.9,
      "severity": "MEDIUM"
    },
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "NETWORK",
        "availabilityImpact": "HIGH",
        "baseScore": 9.8,
        "baseSeverity": "CRITICAL",
        "confidentialityImpact": "HIGH",
        "integrityImpact": "HIGH",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
        "version": "3.1"
      },
      "exploitabilityScore": 3.9,
      "impactScore": 5.9
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
            "versionEndExcluding": "2.2.9",
            "versionStartIncluding": "2.2",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:djangoproject:django:3.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
            "versionEndExcluding": "1.11.27",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      },
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:16.04:*:*:*:lts:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:18.04:*:*:*:lts:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:19.04:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:canonical:ubuntu_linux:19.10:*:*:*:*:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "lastModifiedDate": "2023-11-07T03:07Z",
  "publishedDate": "2019-12-18T19:15Z"
}
